cocos/2d/CCNode.h

```diff
@@ -142,6 +143,15 @@ class CC_DLL Node : public Ref
     /// @{
     /// @name Setters & Getters for Graphic Properties
 
+    /*
+    setZLocalOrderOfArrive totally control the render order, it use for sometime node should cover the friendnode and sometime under the friendnode
+    */
+    void setZLocalOrderOfArrive(std::int64_t nZOrderArrived);
```

Comment:
from 3.9  render seq is ++from a static val and the zorder.
i think it is good idea to control the render seq.
now is only zorder. so if you can apply change child seq to change render seq